### PR TITLE
Some Paint dartdocs (and style tweaks)

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: sky/engine/core/dart/painting.dart [hunk 1]
```diff
@@ -88,24 +90,29 @@ enum BlurStyle {
...
+  MaskFilter.blur(BlurStyle style, double sigma, {
+    bool ignoreTransform: false,
+    bool highQuality: false
+  }) {
     _constructor(style.index, sigma, _makeBlurFlags(ignoreTransform, highQuality));
   }
   void _constructor(int style, double sigma, int flags) native "MaskFilter_constructor";
+
+  // Convert constructor parameters to the SkBlurMaskFilter::BlurFlags type.
+  int _makeBlurFlags(bool ignoreTransform, bool highQuality) {
```
Comment (abarth) [inline sky/engine/core/dart/painting.dart]: static

File: sky/engine/core/dart/painting.dart [hunk 2]
```diff
@@ -169,6 +172,11 @@ class Gradient extends Shader {
     _initRadial(center, radius, colors, colorStops, tileMode.index);
   }
   void _initRadial(Point center, double radius, List<Color> colors, List<double> colorStops, int tileMode) native "Gradient_initRadial";
+
+  void _validateColorStops(List<Color> colors, List<double> colorStops) {
```
Comment (abarth) [inline sky/engine/core/dart/painting.dart]: static

**Comment by abarth** [comment]
LGTM

**Comment by Hixie** [comment]
cc @qchong

**Pull request merged**